radixdlt/babylon-wallet-android
# [ABW-2983] - Update RET to 1.0.12

## Description


## How to test

1. Run sandbox integration tests to verify they execute as expected. 
2. You can use manifest `CALL_METHOD
    Address("account_addr_1")
    "set_resource_preference"
    Address("resource_addr_1")
    Enum<0u8>()
;` sent from sandbox before and after this change to validate that indeed reserved instructions list changed after RET update and now wallet does not reject it

## PR submission checklist
- [ ] I have run integration tests and tested given manifest.
